README.adoc

```diff
@@ -11,27 +11,39 @@
 :page-releasedate: 2018-02-05
 :page-description: Learn how to get started with Bean Validation and Open Liberty.
 :page-tags: ['Java', 'Getting Started']
-:page-related-guides: ['']
 :page-permalink: /guides/{projectid}
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 = Creating an application with Bean Validation
 
-You'll explore how to create an application using Bean Validation 2.0 in Open Liberty.
+You'll explore how to create an application to validate data using Bean Validation in Open Liberty.
 
 // =================================================================================================
 // What you'll learn
 // =================================================================================================
 == What you'll learn
 
-You will learn the basics of writing an application that uses Bean Validation, the new functionality in Bean Validation 2.0, and how to test an application that uses Bean Validation.
+You will learn the basics of writing an application that uses Bean Validation, the new
+functionality in Bean Validation 2.0, and how to test an application that uses Bean Validation. The
+application will allow you to validate data from a form or servlet request using Bean Validation to 
+ensure that it meets the defined constraints.
 
 === What is Bean validation
 
-Bean Validation is a Java Specification allowing data stored in JavaBeans to be validated in a standard way. This can help to simplify validating data and error checking. Validation can be performed manually or via integration with other specifications and frameworks, such as CDI, JPA, or JSF. To set rules on data, constraints are applied using annotations or XML configuration files. Bean Validation provides both built-in constraints and the ability to create custom constraints.
+Bean Validation is a Java specification allowing data stored in JavaBeans to be validated in a
+standard way, simplifying validating data and error checking. Validation can be performed manually
+or via integration with other specifications and frameworks, such as CDI, JPA, or JSF. To set rules
+on data, apply constraints using annotations or XML configuration files. Bean Validation
+provides both built-in constraints and the ability to create custom constraints. Bean Validation 
+allows for validation of both a JavaBean's fields or methods. For method-level validation, both the 
+input parameters and return value can be validated. The application written in this guide will use 
```

Comment:
It seems like the last sentence here, "The application written in this guide will use.." can be on a separate paragraph, because it's not part of "What is Bean validation". It's letting user know about the application they will be developing. Perhaps this new paragraph can go after both the "What is Bean validation" and "What's new in BV 2.0".

Reply:
I've moved that sentence to after "What is Bean validation" and "What's new in BV 2.0".